### PR TITLE
Chore/pull request template

### LINKED ISSUE #2980: Create a checklist for PR reviews
PR reviews are essential for maintaining the quality of our source code. The code reviews should provide another perspective to the code and should check logic of the code as well as readability of the code. The code reviewer must not only check that the code fulfills the goal described in the original issue, but also that the reviewer and other developers would be able to maintain the code later.

Therefore it would be good to document what needs to be inspected in a PR review as a checklist.

### PR DESCRIPTION
**Pull request by mauriciovieira**
Close #2980

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Describe your changes here as a bulleted list:
+- change one
+- change two
+
+## Developer checklist
+This checklist is to be completed by the PR developer:
+
+- [ ] Alternative solutions were compared/discussed before writing code
+    - [ ] trade-offs with this solution are considered acceptable
+- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
```
Comment (brylie) [inline .github/PULL_REQUEST_TEMPLATE.md]: The link to CONTRIBUTING.md may need to be changed.

**Comment by brylie** [comment]
Just a small fix to the CONTRIBUTING.md link needed.

**Comment by bajiat** [comment]
@mauriciovieira Can you modify the link to contributing.md ?

**Comment by mauriciovieira** [comment]
@bajiat this change is not needed. You can [see the generated Markdown](https://github.com/apinf/platform/blob/9f76e0b6fbe558e53cb473e2de6a7a91d718cf23/.github/PULL_REQUEST_TEMPLATE.md) and click on the link

**Pull request merged**